keps/sig-node/4176-cpumanager-spread-cpus-preferred-policy/kep.yaml

```diff
@@ -0,0 +1,42 @@
+title: New CPUManager Static Policy which spread hyperthreads across physical CPUs to better utilize CPU Cache
+kep-number: 4176
+authors:
+  - "@Jeffwan"
+  - "@LingyanYin"
+  - "@horacexd"
+  - "@LastNight1997"
+owning-sig: sig-node
+participating-sigs: []
+status: provisional
```

Comment:
Is this right?

Reply:
no, it should be implementable

Reply:
should I change it from `provisional` to `implementable`?

Reply:
Sounds good. I changed it to `implementable `